test/support/utils_invoke.h

```diff
@@ -219,6 +219,15 @@ struct invoke_on_all_hetero_policies
             auto my_policy = make_new_policy<kernel_name>(queue);
             iterator_invoker<::std::random_access_iterator_tag, /*IsReverse*/ ::std::false_type>()(
                 my_policy, op, ::std::forward<Args>(rest)...);
+
+#if !__SYCL_UNNAMED_LAMBDA__
+            if (std::sin(0) == std::cos(0))
```

Comment:
Some runtime `false` condition check just to compile Kernel code without their run.